components/news/_id/OtherNews.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-row>
+    <v-row
+      justify="space-between"
+    >
+      <v-col
+        cols="auto"
+      >
+        <h2 class="">
+          {{ $t('news.specific_news.other_news') }}
+        </h2>
+      </v-col>
+      <v-col
+        cols="auto"
+      >
+        <v-btn
+          text
```

Comment:
Make it a link to the main news page instead of doing nothing
```suggestion
          text
          nuxt
          to="/news"
```